content/glossary.md

```diff
@@ -33,4 +66,16 @@ draft: true
 
 **Timeboard**
 
-**Trace**
\ No newline at end of file
+**Tracer**
```

Comment:
have we changed the term to tracer from trace?

Reply:
We use them both. A Tracer submits traces.